Include/internal/pycore_interp.h

```diff
@@ -168,8 +171,26 @@ struct _is {
 
     struct ast_state ast;
     struct type_cache type_cache;
+
+    // This must be last.
+    struct {
+        int initialized;
```

Comment:
I don't understand this. What is `initialized`?
If it is the runtime, then why is it in this struct? 
If it is the thread state, then why isn't it in the thread state?

Reply:
That was supposed to be dropped in 51a32ce.